### ISSUE TITLE
Add ability to instrument groovy class files

### ISSUE THREAD

**Issue by jcabraham** [cschabl/activejdbc-gradle-plugin#5]
The instrumentor code looks for java and scala class files (classes/java/main and classes/scala/main) but not for classes in (classes/groovy/main). This is a shame because a) some people still use groovy for UI/DB development, and b) the plugin itself is written in groovy! Will try to create a PR myself for this, but might take a week.

**Comment by cschabl** [comment]
A PR would be welcome.
This plugin, that was part of the ActiveJdbc project formerly, was introduced to provide the same functionality as the Maven plugin for gradle-built projects.
Adding support for Groovy will add value to this plugin.

**Comment by jcabraham** [comment]
Question: I've added the desired functionality, created a test class, all works as expected. I built the plugin (as version 1.3), but I'm having trouble adding it as a local dependency to my project. From the gradle documents, something like this should work

`buildscript {
    dependencies {
        classpath 'de.schablinski.activejdbc-gradle-plugin:1.3'
    }
}`

if the jar  activejdbc-gradle-plugin-1.3.jar is in the project root. However, I get this error: 

`A problem occurred configuring root project 'dsregistry'.
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not find de.schablinski.activejdbc-gradle-plugin:1.3:.
     Required by:
         project :`

I find gradle documentation thorough, yet lacking in what I suppose are common use cases. Any assistance would be appreciated.
